lib/cli.js

```diff
@@ -17,14 +25,18 @@ yargs.usage('\nUsage: contentful <cmd> [args]')
     describe: 'Show current version',
     skipValidation: true
   })
+  .version(false)
```

Comment:
We have our own version implementation to avoid conflict with `contentful space extension update --version`. yargs 9 did enable this by default, so I had to disable it again.